ports/webassembly/library.js

```diff
@@ -33,10 +33,9 @@ mergeInto(LibraryManager.library, {
                 process.stdout.write(b);
             } else {
                 var c = String.fromCharCode(getValue(ptr + i, 'i8'));
-                var mp_js_stdout = document.getElementById('mp_js_stdout');
-                var print = new Event('print');
-                print.data = c;
-                mp_js_stdout.dispatchEvent(print);
+                var printEvent = new Event('micropython-print');
```

Comment:
Should we use [CustomEvent](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) here?

Reply:
Hmm... depends on the context.

> The CustomEvent interface represents events initialized by an application for any purpose.

(from MDN)

Is MicroPython an "application"..? My intuition is `CustomEvent` is more appropriate from a purely semantic point of view.

Reply:
Shall we use `CustomEvent` then?

Reply:
Yes. That's the semantically correct object to use. I'll change/push.